src/MizarWidgets/SamplingWithFrameTrackInputWidget.ui

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>SamplingWithFrameTrackInputWidget</class>
+ <widget class="QWidget" name="SamplingWithFrameTrackInputWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>569</width>
+    <height>233</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <layout class="QGridLayout" name="gridLayout">
+   <item row="0" column="0">
+    <layout class="QVBoxLayout" name="main_layout_">
+     <item>
+      <layout class="QHBoxLayout" name="header_layout_">
+       <item>
+        <widget class="QLabel" name="title_">
+         <property name="minimumSize">
+          <size>
+           <width>90</width>
+           <height>0</height>
+          </size>
+         </property>
+         <property name="maximumSize">
+          <size>
+           <width>90</width>
+           <height>16777215</height>
+          </size>
+         </property>
+         <property name="text">
+          <string>NONAME</string>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QLabel" name="label_2">
+         <property name="maximumSize">
+          <size>
+           <width>70</width>
+           <height>16777215</height>
+          </size>
+         </property>
+         <property name="text">
+          <string>From ms:</string>
```

Comment:
Still not 100% sure, if this is descriptive enough. But maybe its fine.

Reply:
Tooltips are good. But not in this PR.

Reply:
Would be fine with me. But note that you can simply add a tooltip in the QtDesigner.